sdks/python/apache_beam/runners/common.py

```diff
@@ -402,7 +402,7 @@ def __init__(self,
     self.window_fn = windowing.windowfn
 
     self.do_fn_invoker = DoFnInvoker.create_invoker(
-        do_fn_signature, context, side_inputs, args, kwargs)
+        self, do_fn_signature, context, side_inputs, args, kwargs)
```

Comment:
can we pass just the _process_output function instead of self so that dependent class doesn't need to know about _process_output function